packages/workspace/migrations.json

```diff
@@ -293,6 +298,55 @@
           "alwaysAddToPackageJson": false
         }
       }
+    },
+    "8120": {
+      "version": "8.12.0",
+      "packages": {
+        "@angular-devkit/architect": {
+          "version": "0.803.22",
+          "alwaysAddToPackageJson": false
+        },
+        "@angular-devkit/build-angular": {
+          "version": "0.803.22",
+          "alwaysAddToPackageJson": false
+        },
+        "@angular-devkit/build-ng-packagr": {
+          "version": "0.803.22",
+          "alwaysAddToPackageJson": false
+        },
+        "@angular-devkit/build-webpack": {
+          "version": "0.803.22",
+          "alwaysAddToPackageJson": false
+        },
+        "@angular-devkit/build-optimizer": {
+          "version": "0.803.22",
+          "alwaysAddToPackageJson": false
+        },
+        "@angular-devkit/core": {
+          "version": "8.3.22",
+          "alwaysAddToPackageJson": false
+        },
+        "@angular-devkit/schematics": {
+          "version": "8.3.22",
+          "alwaysAddToPackageJson": false
+        },
+        "@angular/cli": {
+          "version": "8.3.22",
+          "alwaysAddToPackageJson": false
+        },
+        "@schematics/angular": {
+          "version": "8.3.22",
+          "alwaysAddToPackageJson": false
+        },
+        "browserslist": {
```

Comment:
I wonder whether `browserslist` and `caniuse-lite` is actually needed in the migrations 🤔

Reply:
They are only needed if they are present in the package.json file of the workspace. So I'd say no. 

`@angular-devkit/core` technically doesn't have to be there either.